config/routes.rb

```diff
@@ -26,10 +22,18 @@
 
   resources :users
 
-  resources :reservations
-
   resources :admins
 
+  namespace :user, path: "/:slug" do
+    resources :listings, except: [:index]
+    get '/dashboard', to: 'dashboard#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.